### ISSUE TITLE
Nuitka 4.0 unexpected result type <class 'dict'> for standard.nuitka-package.config.yml module torch.utils._config_module

### ISSUE THREAD

**Issue by E-waal** [Nuitka/Nuitka#3760]
# Bug Report

______________________________________________________________________

## ⚠️ Pre-Submission Checklist

*Please check these items before submitting your report. This text can be removed.*

- [ ] **Search Existing Issues:** Has this bug already been reported or fixed? Sometimes reports you
  find are bad, always feel free to make a better one, but reference what you found.
- [ ] **Python vs. Nuitka:** Does the issue occur when running the code with standard Python? If
  yes, it's not a Nuitka-specific issue, do not report it.
- [ ] **Test `develop` Branch:** Have you tried the `develop` version of Nuitka? It might contain a
  fix. Get it here https://nuitka.net/doc/download.html

______________________________________________________________________

**Discard everything above this line before submitting your issue.**

## 🐛 Bug Description

*Nuitka 4.0 produces an error when compiling torch*

### 1. Nuitka Version, Python Version, OS, and Platform

(ifu) C:\Users\Utilisateur\Desktop\ISAC>python -m nuitka --version
4.0
Commercial: None
Python: 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Flavor: CPython Official
Executable: ~\Desktop\ISAC\ifu\Scripts\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: gcc (gcc 14.2.0).

### 2. How Nuitka and Python were Installed

*pip*

### 3. Relevant PyPI Packages and Versions

Package                   Version
------------------------- ------------
altgraph                  0.17.5
annotated-doc             0.0.4
annotated-types           0.7.0
anyio                     4.12.1
arrow                     1.4.0
beautifulsoup4            4.14.3
binaryornot               0.4.4
certifi                   2026.1.4
cffi                      2.0.0
chardet                   5.2.0
charset-normalizer        3.4.4
click                     8.3.1
colorama                  0.4.6
contourpy                 1.3.2
cookiecutter              2.6.0
cryptography              46.0.3
cycler                    0.12.1
exceptiongroup            1.3.1
fastapi                   0.128.0
filelock                  3.20.3
flet                      0.27.4
flet-cli                  0.27.4
flet-desktop              0.27.4
flet-web                  0.27.4
fonttools                 4.61.1
fsspec                    2026.1.0
h11                       0.16.0
httpcore                  1.0.9
httptools                 0.7.1
httpx                     0.28.1
idna                      3.11
Jinja2                    3.1.6
kiwisolver                1.4.9
lap                       0.5.12
markdown-it-py            4.0.0
MarkupSafe                3.0.3
matplotlib                3.10.8
mdurl                     0.1.2
mpmath                    1.3.0
networkx                  3.4.2
Nuitka                    4.0
numpy                     2.2.6
oauthlib                  3.3.1
opencv-python             4.13.0.90
ordered-set               4.1.0
packaging                 25.0
pandas                    2.3.3
pdf2image                 1.17.0
pdfminer.six              20251230
pdfplumber                0.11.9
pefile                    2024.8.26
pillow                    12.1.0
pip                       23.0.1
polars                    1.37.1
polars-runtime-32         1.37.1
psutil                    7.2.1
pycparser                 3.0
pydantic                  2.12.5
pydantic_core             2.41.5
Pygments                  2.19.2
pyinstaller               6.18.0
pyinstaller-hooks-contrib 2025.11
PyMuPDF                   1.26.7
pyparsing                 3.3.1
pypdfium2                 5.3.0
pypng                     0.20220715.0
python-dateutil           2.9.0.post0
python-dotenv             1.2.1
python-slugify            8.0.4
pytz                      2025.2
pywin32                   311
pywin32-ctypes            0.2.3
PyYAML                    6.0.3
pyzbar                    0.1.9
qrcode                    7.4.2
qrdet                     2.5
qreader                   3.16
quadrilateral-fitter      1.12
regex                     2026.1.15
repath                    0.9.0
requests                  2.32.5
rich                      14.2.0
schedule                  1.2.2
scipy                     1.15.3
setuptools                65.5.0
shapely                   2.1.2
six                       1.17.0
soupsieve                 2.8.2
starlette                 0.50.0
sympy                     1.14.0
text-unidecode            1.3
toml                      0.10.2
torch                     2.8.0
torchaudio                2.8.0
torchvision               0.23.0
tqdm                      4.67.1
typing_extensions         4.15.0
typing-inspection         0.4.2
tzdata                    2025.3
ultralytics               8.4.6
ultralytics-thop          2.0.18
urllib3                   2.6.3
uvicorn                   0.40.0
watchdog                  4.0.2
watchfiles                1.1.1
websockets                16.0
zstandard                 0.25.0

### 1. "Hello World" Test (if applicable)

If you suspect a setup issue, first try compiling a simple `print("hello world")` program. If that
fails, report that error instead. *(We don't want a "Tensorflow does not work" report if basic
compilation fails. Please also consult the User Manual for supported setups.)*

### 2. Short, Self-Contained, Correct, Eligible (SSCCE) Example

Provide a minimal code example that reproduces the issue. It should have as few external
dependencies as possible. *(Issues without a clear, runnable example are likely to be rejected. For
packaging issues, a simple import might suffice. Repositories with `pipenv` or archives with small
examples are very welcome, but must contain only source code. Ensure usage is obvious; a compile
script is ideal. For non-commercial users, keep the code brief.)*

```python
from qreader import QReader
import cv2

qreader = QReader()
image = cv2.imread("path/to/image.png")
decoded_text = qreader.detect_and_decode(image=image)
print(decoded_text)
```

### 3. Nuitka Command Line Options

```python
python -m nuitka --standalone --enable-plugin=tk-inter --module-parameter=torch-disable-jit=yes script.py
```

**Important Notes on Options:**

- **Avoid all unnecessary options:**
  - Do **not** use `--deployment`. It disables bug-catching features. Remove it if you encounter an
    issue.
  - If the issue occurs with `--mode=standalone`, do not report it with `--mode=onefile` unless the
    problem is specific to onefile. State if the issue also occurs in standalone mode if you report
    a onefile issue.
  - Minimize options used. Do not use `--quiet` or disable warnings.

## 📉 Expected Behavior

*Compile without issue*

## 📄 Actual Behavior & Output

*FATAL: Error, unexpected result type <class 'dict'> for standard.nuitka-package.config.yml module torch.utils._config_module section constants.*

## ↩️ Regression (if applicable)

Work well on Nuitka 2.8.10

## 💡 Additional Context (Optional)

*(Add any other context about the problem here.)*

______________________________________________________________________

## 🚫 Unwelcome Practices (Please Read)

- **No Screenshots of Text:** Do *not* post screenshots of code, errors, or console output unless
  absolutely necessary (e.g., a GUI display problem). Provide text directly so it's searchable and
  copyable. Reports without proper text are likely to be rejected.
- **Do Not Close Issues Yourself:** We will close issues once they are resolved in a stable release.
  If you find your issue is invalid, you may close it, and we will tag it as `invalid` or even
  delete it later.
- **Do Not Report Against `factory` Branch:** Unless specifically asked, avoid reporting issues
  found on the `factory` branch as it's often unstable. Discuss on Discord first.

______________________________________________________________________

## 💼 Commercial Support

Consider Nuitka commercial support for priority assistance, which also helps sustain Nuitka's
development. This allows for direct code sharing or paid time for issue resolution in your
environment.

______________________________________________________________________

**PLEASE REMOVE ALL TEMPLATE TEXT (INCLUDING THIS LINE) BEFORE SUBMITTING YOUR ISSUE.**

**Comment by Doitzh** [comment]
I'm currently stuck in this BUG.

**Comment by akashlevy** [comment]
+1

**Comment by kayhayen** [comment]
Thanks for your report. This is fixed on the factory branch, which is a development version that is under rapid development. You can try it out by going here: https://nuitka.net/doc/factory.html 
	
Feedback on whether this is working is very welcome. Please do not share plans to do it; only confirm or deny that it is working.

**Comment by E-waal** [comment]
The factory branch didn't work. Still the same Bug.

**Comment by BenCzaczkes** [comment]
Same here on a 5-line script. The factory branch didn't work
on a clean new .venv with only torch installed

**Comment by kayhayen** [comment]
I think I have forgotten to actually push factory, let me know if this is still and issue with 4.0.1, sorry for the confusion, or maybe I am all wrong.
